src/ucp/proto/proto.h

```diff
@@ -42,6 +42,15 @@ typedef unsigned ucp_proto_id_t;
 typedef uint64_t ucp_proto_id_mask_t;
 
 
+/**
+ * Protocol flags for internal usage, to allow searching for specific protocols
+ */
+enum {
+    UCP_PROTO_FLAG_AM_SHORT  = UCS_BIT(0), /* The protocol uses only uct_ep_am_short() */
+    UCP_PROTO_FLAG_PUT_SHORT = UCS_BIT(0)  /* The protocol uses only uct_ep_put_short() */
```

Comment:
the value is the same as `UCP_PROTO_FLAG_AM_SHORT` intentionally, right?
So there will be names for every API which support certain protocol?

Reply:
sorry that was a typo, fixed